action_plugins/juniper_junos_file.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this action plugin is the same as juniper_junos_common_action.py. If it is, it should be a link to juniper_junos_common_action.py rather than a copy/paste of the file. The diff doesn't make it clear if this is a file or a link, so if it's already a link, please ignore my comment.